### ISSUE TITLE
inputScanner function has too many processes | TODO #7

### ISSUE THREAD

**Issue by Abid-Chowdhury** [Abid-Chowdhury/Lab-2#2]
### 14 processes are being done inside the inputScanner function

1. create a input data list
2. initialize scanner to get user input
3. get account number
4. check if customer account number is valid
5. get customer code
6. ask for input -> convert input to string
7. check if customer code is valid
8. get number of premium channels
9. check if premium channels input is valid
10. get number of connections (if business customer)
11. check if connections input is valid
12. close the scanner
13. add user input to list
14. return inputData

**TASK**
Break down inputScanner function into smaller functions

**POSSIBLE SOLUTION**
```
main()
    getInput()
        initialize connections
        initialize customerCode
        displayCompanyTitle()
        inputScanner()
            createCustomerClass()
                create a customer class
                    attributes
                        account number
                        customer code
                        premium channels
                        connections

                    constructer
                        account number
                        customer code
                        premium channels
                        connections

                    methods
                        // setters
                        setAccountNumber()
                            sets account number
                        setCustomerCode()
                            sets customer code
                        setPremiumChannels()
                            sets premium channels
                        setConnections()
                            sets connections

                        // getters
                        getAll()
                            return all data
                        getAccountNumber()
                            return account number
                        getCustomerCode()
                            return customer code
                        getPremiumChannels()
                            return premium channels
                        getConnections()
                            return connections

            getInputData()
                initialize scanner to get user input
                getAccountNumber()
                    checkIfAccountNumberValid()
                    customer.setAccountNumber()
                getCustomerCode()
                    ask for input
                    convert input to string
                    checkIfCustomerCodeValid()
                    customer.setCustomerCode()
                get number of premium channels()
                    checkIfPremiumChannelsValid()
                    customer.setPremiumChannels()
                if (businessCustomer)
                {
                    getNumberOfConnections()
                    checkIfNumberOfConnectionsValid()
                    customer.setConnections()
                }
                close scanner

            return customer object

```

**Comment by Abid-Chowdhury** [comment]
Fixed in 88fcf787d4e7382ea28eba833caf60bab011f9bc

**Comment by Abid-Chowdhury** [comment]
Fixed in 88fcf787d4e7382ea28eba833caf60bab011f9bc
